cmd/agent/app/dependent_services.go

```diff
@@ -0,0 +1,35 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2017 Datadog, Inc.
+
+package app
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/config"
+	log "github.com/cihub/seelog"
+	
+)
+
+// start various subservices (apm, logs, process) based on the config file settings
+
+// checks to see if a given service should be started
+func (s* Servicedef) IsEnabled() bool {
+	return config.Datadog.GetBool(s.configKey)
+}
+
+func startDependentServices()  {
+	for _, svc := range subservices {
+		if svc.IsEnabled() {
+			log.Debugf("Attempting to start service: %s", svc.name)
+			err := svc.Start()
+			if err != nil {
+				log.Warnf("Failed to start services %s: %s", svc.name, err.Error())
+			} else {
+				log.Debugf("Started service %s", svc.name)
+			}
+		} else {
+			log.Infof("Service %s is disabled, not starting", svc.name)
```

Comment:
all of these log lines would also be logged on Unix, which would be confusing. Since the external agents aren't managed in the same way at all on Unix (at least presently), how about moving all the logic of `startDependentServices` to the windows-only file, and having the unix file basically empty? (except an empty `startDependentServices` function)